src/main/java/tools/map/xml/creator/UnitPlacementsPanel.java

```diff
@@ -22,7 +22,6 @@
 import java.util.Map.Entry;
 import java.util.Optional;
 import java.util.stream.Collectors;
-
```

Comment:
@DanVanAtta Just a heads up...  I didn't notice this until after syncing up with some of your recent PRs: I think there is a divergence between the IntelliJ and Eclipse import order settings.  If I run Organize Imports with Eclipse, it will split `java.*` and `javax.*` into different groups, but it looks like the IntelliJ settings merge them into a single group.

I'm not suggesting you go back and change the instances where these import groups have been merged.  But it would be good to get the settings consistent between the two IDEs so we don't ping-pong between the two styles as we keep checking in changes.  I'm not sure if [this information](https://github.com/triplea-game/triplea/pull/1589#issuecomment-290617240) is still up-to-date, but I seem to remember it working when I tested it (although now I'm not sure if I tried testing a file that had `javax.*` packages).